examples/python-django/devenv.nix

```diff
@@ -0,0 +1,79 @@
+{ pkgs, ... }:
+
+let
+  db_user = "postgres";
+  db_host = "localhost";
+  db_port = "5432";
+  db_name = "db";
+in
+{
+  packages = [ pkgs.git pkgs.postgresql_14 ];
+
+  languages.python = {
+    enable = true;
+    package = pkgs.python310;
+    poetry.enable = true;
+  };
+
+  env = {
+    DATABASE_URL = "postgresql://${db_user}@${db_host}:${db_port}/${db_name}";
```

Comment:
It would be better to use `env.PGHOST` that points to the unix socket, so that multiple projects can be launched in parallel.